appendices/VK_EXT_physical_device_drm.txt

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020 Simon Ser
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+include::{generated}/meta/{refprefix}VK_EXT_physical_device_drm.txt[]
+
+=== Other Extension Metadata
+
+*Last Modified Date*::
+    2021-06-09
+*IP Status*::
+    No known IP claims.
+*Contributors*::
+    - Simon Ser
+
+=== Description
+
+This extension provides new facilities to query DRM properties for physical
+devices, enabling users to match Vulkan physical devices with DRM nodes on
+Linux.
+
+Its functionality closely overlaps with
+`EGL_EXT_device_drm`^<<VK_EXT_physical_device_drm-fn1,1>>^.
+Unlike the EGL extension, this extension does not expose a string containing
+the name of the device file and instead exposes device minor numbers.
+
+DRM defines multiple device node types. Each physical device may have one
+primary node and one render node associated. Physical devices may have no
+primary node (e.g. if the device doesn't have a display subsystem), may have
+no render node (e.g. if it's a software rendering engine), or may have neither
+(e.g. if it's a software rendering engine without a display subsystem).
+
+To query DRM properties for a physical device, chain
+slink:VkPhysicalDeviceDrmPropertiesEXT to slink:VkPhysicalDeviceProperties2.
+
+=== References
+
+  . [[VK_EXT_physical_device_drm-fn1]]
+    https://www.khronos.org/registry/EGL/extensions/EXT/EGL_EXT_device_drm.txt[`EGL_EXT_device_drm`]
+
```

Comment:
The extension overview should include the standard list of new types, entry points, enums, etc added by the extension. The information is autogenerated from the XML by the build chain, so all you need to do is insert this line here:

```suggestion
include::{generated}/interfaces/VK_EXT_physical_device_drm.txt[]
```

Reply:
Done!